src/page-response-impact/styles.tsx

```diff
@@ -83,7 +83,7 @@ export const ReportDateDiv = styled.div`
 `;
 
 export const DescriptionTextDiv = styled.div`
-  font-family: Helvetica Neue;
+  font-family:  "Poppins", sans-serif;
```

Comment:
Ultra nit: an extra space after the colon here.

Reply:
👍 fixed it